src/bin/trace-verifier/commands/run_file.rs

```diff
@@ -37,10 +37,13 @@ impl RunFileCommand {
             })?;
 
             let fork_config = fork_config(trace.chain_id);
-            tokio::task::spawn_blocking(move || {
+            if let Err(err) = tokio::task::spawn_blocking(move || {
                 utils::verify(trace, &fork_config, disable_checks, false)
             })
-            .await??;
+            .await?
+            {
+                dev_error!("Verification failed: {:?}", err);
```

Comment:
The `dev_error` here is not necessary. The guest program in `sproll-evm` depends only on the library from `stateless_block_verifier`. This binary `trace-verifier` is just a helper for us to test SBV. So we can remove this.

Instead, similar to the `run_rpc.rs` you should do:
```rust
let mut guard = error_log.as_ref().unwrap().lock().await;
guard.write_all(/* */).await?;
```